header.php

```diff
@@ -46,7 +46,24 @@
         </ul>
       </nav>
     </div>
+   </div>
 
 <!--  <img src="http://Setupshop/wp-content/uploads/2016/09/SA.png" alt="sa-flags">  -->
 
-<?php dynamic_sidebar( 'Filters' ); ?>
\ No newline at end of file
+<!--<?php dynamic_sidebar( 'Filters' ); ?>-->
```

Comment:
When you're no longer using code, don't forget to delete. And if you're commenting out php it's easier to do this within the tags so it doesn't get executed.